src/unix/osd.c

```diff
@@ -120,6 +120,26 @@ int ofi_wait_cond(pthread_cond_t *cond, pthread_mutex_t *mut, int timeout_ms)
 	return pthread_cond_timedwait(cond, mut, &ts);
 }
 
+int ofi_mmap_anon_pages(void **memptr, size_t size)
+{
+	size = ofi_get_aligned_size(size, ofi_get_page_size());
```

Comment:
Should we return the corrected size to caller?

Reply:
New revision calculates the size outside of this function.